model/model.go

```diff
@@ -14,18 +14,18 @@ import (
 
 // RouterConfig is the primary type used to encapsulate all router configuration.
 type RouterConfig struct {
-	WorkerProcesses          string      `json:"workerProcesses"`
-	MaxWorkerConnections     int         `json:"maxWorkerConnections"`
-	DefaultTimeout           int         `json:"defaultTimeout"`
-	ServerNameHashMaxSize    int         `json:"serverNameHashMaxSize"`
-	ServerNameHashBucketSize int         `json:"serverNameHashBucketSize"`
-	GzipConfig               *GzipConfig `json:"gzipConfig"`
-	BodySize                 int         `json:"bodySize"`
-	ProxyRealIPCIDR          string      `json:"proxyRealIpCidr"`
-	ErrorLogLevel            string      `json:"errorLogLevel"`
-	Domain                   string      `json:"domain"`
-	UseProxyProtocol         bool        `json:"useProxyProtocol"`
-	EnforceWhitelists        bool        `json:"enforceWhitelists"`
+	WorkerProcesses          string      `router:"workerProcesses"`
```

Comment:
It should be noted that `router` in the struct tags is in no way tied to the annotation keys in the manifests.  And the latter are namespaces nicely like `router.deis.io/whatever`